### DISCUSSION TITLE
Documentation detail

### DISCUSSION THREAD

**Discussion by gunterl** [ESCOMP/CTSM#2910] [Ideas]
This item came up when creating land model diagnostics. It would be great to specify somewhere in the documentation that units regarding snow and ice are outputted in water equivalent (compared to some model who provide output in ice equivalent).

Thanks a lot.

**Comment by samsrabin** [comment]
Good idea, Gunter. I've made an issue for this (#2911).

**Comment by ekluzek** [comment]
Closing since it was made into an issue.

**Comment by samsrabin** [comment]
@gunterl This docs update will come in sometime in the next couple of weeks thanks to #3247.

**Comment by gunterl** [comment]
Hi Sam,

Awesome, thanks a lot.

Gunter

On Thu, Jun 12, 2025 at 12:00 PM Sam Rabin ***@***.***> wrote:

> @gunterl <https://github.com/gunterl> This docs update will come in
> sometime in the next couple of weeks thanks to #3247
> <https://github.com/ESCOMP/CTSM/pull/3247>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ESCOMP/CTSM/discussions/2910#discussioncomment-13451912>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6QV6B6TMMCJSENYT5IWLT3DG545AVCNFSM6AAAAABTXSUPR2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGNBVGE4TCMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Gunter Leguy, Ph.D
Project Scientist
NSF National Center for Atmospheric Research
cell: (575) 418 1021
desk: (303) 497 1790
